app/models/patient.rb

```diff
@@ -101,10 +110,11 @@ class Patient
             :initial_call_date,
             :created_by_id,
             :line,
-            presence: true
+            presence: true, unless: :archived?
```

Comment:
I'd suggest break up this presence check into two, one for phone/name (which we can scramble on archive) with the `unless: :archived?` and one for the rest. I figure we're always going to want to keep initial call date, line, etc around.